src/neo/SmartContract/ApplicationEngine.OpCodePrices.cs

```diff
@@ -181,11 +181,11 @@ partial class ApplicationEngine
             [OpCode.SIZE] = 150,
             [OpCode.HASKEY] = 270000,
             [OpCode.KEYS] = 500,
-            [OpCode.VALUES] = 7000,
+            [OpCode.VALUES] = 270000,
             [OpCode.PICKITEM] = 270000,
-            [OpCode.APPEND] = 15000,
+            [OpCode.APPEND] = 270000,
             [OpCode.SETITEM] = 270000,
-            [OpCode.REVERSEITEMS] = 500,
+            [OpCode.REVERSEITEMS] = 270000,
```

Comment:
Why it cost so much?

Reply:
Because reversing a 1024*1024-sized buffer would be time consuming (about 500 μs per construction) and can be called for duplicately.

Reply:
Do you have the test case?

Reply:
As follows:

```
using (var engine = new ExecutionEngine())
            using (var script = new ScriptBuilder())
            {
                script.EmitPush(1024 * 1024);
                script.Emit(OpCode.NEWBUFFER);
                script.Emit(OpCode.REVERSEITEMS);
                engine.LoadScript(script.ToArray());
                engine.Execute();
            }
```